web/public/images/star-unfilled-bright.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M22.8394 9.65442L22.8394 9.65445L22.8474 9.65487C22.8606 9.65557 22.8678 9.65743 22.8704 9.65825C22.8718 9.66003 22.8746 9.6642 22.877 9.67159C22.881 9.68367 22.8796 9.69173 22.8792 9.69294L22.8792 9.69297C22.8792 9.69307 22.8792 9.69329 22.8786 9.69423C22.8781 9.69503 22.8743 9.70092 22.8628 9.7105C22.8626 9.71062 22.8625 9.71074 22.8624 9.71086L17.7498 13.9256C17.1086 14.4285 16.8775 15.2182 17.0194 15.9274L17.0251 15.9561L17.0325 15.9844L18.7055 22.3766C18.7082 22.3885 18.7078 22.3942 18.7078 22.3949L18.7076 22.3961C18.7071 22.3973 18.7035 22.4046 18.6933 22.4121C18.6856 22.4177 18.68 22.4192 18.6783 22.4196C18.6782 22.4196 18.6782 22.4195 18.6781 22.4195C18.6772 22.4193 18.6702 22.4175 18.6572 22.4097L13.0761 18.8467C12.3995 18.3891 11.5448 18.4154 10.8842 18.84C10.8842 18.84 10.8842 18.8401 10.8842 18.8401L5.31716 22.41C5.30449 22.4175 5.29769 22.4193 5.29679 22.4195C5.29671 22.4195 5.29664 22.4196 5.29658 22.4196C5.29488 22.4192 5.28927 22.4177 5.28157 22.4121C5.27137 22.4046 5.26776 22.3973 5.26729 22.3961L5.2671 22.3949C5.26705 22.3942 5.26667 22.3884 5.26946 22.3763L6.94117 15.9639C6.94144 15.9629 6.94172 15.9618 6.94199 15.9608C7.14836 15.1817 6.83996 14.3671 6.26512 13.888L6.26514 13.888L6.25879 13.8828L1.1375 9.68586C1.13716 9.68558 1.13683 9.6853 1.13651 9.68503C1.12545 9.67573 1.1218 9.67002 1.12129 9.66923C1.12069 9.66829 1.12065 9.66807 1.12063 9.66797L1.12062 9.66794C1.1203 9.66673 1.11891 9.65867 1.12284 9.64659C1.12524 9.63919 1.12808 9.63503 1.12948 9.63325C1.13208 9.63243 1.13924 9.63056 1.15249 9.62987L1.15249 9.6299L1.1602 9.62943L7.7852 9.22943L7.78731 9.2293C8.56913 9.18044 9.2586 8.68967 9.55341 7.95264L9.55343 7.95265L9.55573 7.94679L11.9791 1.77601C11.9848 1.76208 11.9894 1.75646 11.9902 1.75552L11.9909 1.75471L11.9911 1.75449L11.9913 1.75436C11.992 1.75392 11.999 1.75 12.0124 1.75C12.0258 1.75 12.0329 1.75392 12.0336 1.75436L12.0338 1.75449L12.034 1.75471L12.0347 1.75552C12.0355 1.75645 12.04 1.76199 12.0457 1.77567L14.4686 7.97051L14.4686 7.97052L14.4715 7.97764C14.7663 8.71467 15.4557 9.20544 16.2376 9.2543L16.2394 9.25442L22.8394 9.65442Z" stroke="#C6C6C6" stroke-width="2"/>
```

Comment:
I think this colour code works for the sidebar, but we might want a `#FFFFFF` version for the pipeline tiles and top bar favorite buttons in the pipeline view.

Maybe we should think about our embed strategy so that we can adjust background-color and stroke with css instead?

Reply:
yeah i'll take a look into this, having multiple svgs with just colour changes is kind of annoying

Reply:
i think it'd need a bit more thinking, as we're showing those svg's as background images, and i'm not sure if you can just change the stroke of a background image'd svg.

i did find this:
https://css-tricks.com/solved-with-css-colorizing-svg-backgrounds/
could use some filter magic, but i don't think that's a very good solution right now

i'll take tomorrow to see how much work it is to pull them out from bg-image... otherwise perhaps another asset shall be created.

Reply:
@matthewpereira we use background-image everywhere... maybe down the road we could re-look into this, but i think for now another asset might be the way to go to finish this issue

Reply:
Multiple assets is the way to go for now, then, you're right. LMK if you'd like help making the alternate SVG - it's just a matter of duplicating the file and changing the hex codes, but in case you run into anything unexpected I'm here to help.

Reply:
@matthewpereira let me know how it looks, i added a new icon but i think i'll look at it again tomorrow/next week as i'm not super satisfied with the logic!